jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFAnnotationProcessor.java

```diff
@@ -50,6 +50,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import org.jboss.jandex.AnnotationValue;
```

Comment:
We keep imports sorted to avoid unnecessary conflicts, see where `org.jboss...` is imported above.

Reply:
Thanks @rhusar, I'll move it upward tomorrow.

Reply:
Done!